index.html

```diff
@@ -0,0 +1,132 @@
+                    </div>
+                </div>
+
+            </div>
+
+            <div x-show="file" class="my-4 flex justify-end items-center">
+
+                <p><code class="mx-4 text-sm" x-text="getOffestsName(file?.name ?? '')"></code></p>
+
+                <span class="inline-flex rounded-md shadow-sm">
+                  <a
+                    :class="processing ? 'from-pink-500 to-yellow-500 cursor-not-allowed' : 'from-green-400 to-blue-500'"
+                    class="inline-flex items-center justify-between py-2 px-4 border-transparent text-base leading-6 font-medium rounded-md text-white bg-gradient-to-r  transition ease-in-out duration-150"
+                    x-effect="$el.setAttribute('href', URL.createObjectURL(new Blob([JSON.stringify(offsets)], {type: 'text/json'}))); $el.setAttribute('download', getOffestsName(file?.name ?? ''))"
+                  >
+                    <svg :class="processing ? 'animate-spin' : ''" class="-ml-1 mr-3 h-5 w-5 text-white" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
+                      <circle x-show="processing" class="opacity-25" cx="12" cy="12" r="10" stroke="currentColor" stroke-width="4"></circle>
+                      <path x-show="processing" class="opacity-75" fill="currentColor" d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"></path>
+                      <path x-show="!processing" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M7 16a4 4 0 01-.88-7.903A5 5 0 1115.9 6L16 6a5 5 0 011 9.9M9 19l3 3m0 0l3-3m-3 3V10" />
+                    </svg>
+                    <span x-text="processing ? 'Processing' : 'Download'"></span>
+                  </a>
+                </span>
+
+            </div>
+
+            <div x-show="warning" class="m-10 bg-red-100 text-red-700 px-4 py-3 rounded relative" role="alert">
+              <strong class="font-bold">Oop!</strong>
+              <span class="block sm:inline">Something went wrong. Open the console for more information.</span>
+
+              <div class="absolute top-0 bottom-0 right-0 px-4 py-3">
+                <button type="button" @click="warning = false" class="-mr-1 flex p-1 rounded-md sm:-mr-2">
+                  <span class="sr-only">Dismiss</span>
+                  <svg class="h-4 w-4" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor" aria-hidden="true">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+                  </svg>
+                </button>
+              </div>
+
+            </div>
+
+          </div>
+
+      </div>
+  </body>
+  <script>
+      function getFile(e) {
+        let { files } = e.type === 'change' ? e.target : e.dataTransfer;
+        return files[0];
+      }
+      function getOffestsName(name) {
+        let re = /(\.ome?\.tiff?)/;
```

Comment:
I think this regex should have parentheses around the `ome` so that it is optional.  This library can work for any TIFF - the `ome` is just for other tools to recognize that which might need to know that the file is OME

Reply:
If you look at the python code, the `ome` is optional